blaze/__init__.py

```diff
@@ -76,29 +76,8 @@
 with ignoring(ImportError):
     from .sql import *
     from .compute.sql import *
-with ignoring(ImportError):
-    from .compute.dask import *
-with ignoring(ImportError, AttributeError):
-    from .compute.spark import *
-with ignoring(ImportError, TypeError):
-    from .compute.sparksql import *
-with ignoring(ImportError):
-    from .compute.h5py import *
-with ignoring(ImportError):
-    from .compute.hdfstore import *
-with ignoring(ImportError):
-    from .compute.pytables import *
 with ignoring(ImportError):
     from .compute.chunks import *
-with ignoring(ImportError):
-    from .compute.bcolz import *
-with ignoring(ImportError):
-    from .mongo import *
-    from .compute.mongo import *
-with ignoring(ImportError):
-    from .pytables import *
-    from .compute.pytables import *
```

Comment:
This probably improves things enormously wrt import times:
```python
In [1]: %time import blaze as bz
Wall time: 4.6 s
```

I had wondered if some sort of lazy import mechanism might be useful, but with this change perhaps not?

If there's a plugin mechanism for backends though it may still be useful so users don't have to pay for what they're not using.

Reply:
I had thought to use the [`setuptools` plugin capability](http://setuptools.readthedocs.io/en/latest/setuptools.html#dynamic-discovery-of-services-and-plugins) but your solution is simpler and more explicit!